Balloonduino.ino

```diff
@@ -1,1196 +1,40 @@
-#include <avr/wdt.h> // watchdog timer
-#include <Wire.h>
-#include <SPI.h>
-#include <Adafruit_Sensor.h>
-#include <Adafruit_BNO055.h>
-#include <utility/imumaths.h>
-#include <Adafruit_MCP9808.h>
-#include <RTClib.h> // RTC and SoftRTC
-#include <Adafruit_BME280.h>
-#include <SD.h>
-#include <Adafruit_ADS1015.h>
-#include <CCSDS.h>
-#include <ccsds_xbee.h>
-#include <ccsds_util.h>
-#include <SSC.h>
+#include <Balloonduino.h>
 
-#define mars_2
-
-// Program Specific Constants
-
-#ifdef mars_1
-
-#define timer 360 // seconds after last keepalive when actuator will retract
-
-#define ACTUATOR_CONTROL_PIN 2
-
-//// Xbee setup parameters
-#define XBEE_ADDR 0x0004 // XBee address for this payload
-// DO NOT CHANGE without making corresponding change in ground system definitions
-
-/* APIDs */
-#define COMMAND_APID 400
-#define STATUS_APID 401
-#define PACKET_COUNTER_APID 410
-#define ENVIRONMENTAL_PACKET_APID 420
-#define POWER_PACKET_APID 430
-#define IMU_PACKET_APID 440
-
-#endif
-
-#ifdef mars_2
-
-#define timer 180 // seconds after last keepalive when actuator will retract
-
-#define ACTUATOR_PIN_HBRIDGE_A 3
-#define ACTUATOR_PIN_HBRIDGE_B 4
-
-//// Xbee setup parameters
-#define XBEE_ADDR 0x0005 // XBee address for this payload
-// DO NOT CHANGE without making corresponding change in ground system definitions
-
-/* APIDs */
-#define COMMAND_APID 500
-#define STATUS_APID 501
-#define PACKET_COUNTER_APID 510
-#define ENVIRONMENTAL_PACKET_APID 520
-#define POWER_PACKET_APID 530
-#define IMU_PACKET_APID 540
-
-#endif
-
-#define CYCLE_DELAY 100 // time between execution cycles [ms]
-bool extended = false;
-bool armed = true;
-
-//// Enumerations
-// logging flag
-#define LOG_RCVD 1
-#define LOG_SEND 0
-
-/* function codes */
-#define COMMAND_RETRACT_ACTUATOR 01
-#define COMMAND_EXTEND_ACTUATOR 02
-#define REQUEST_ACTUATOR_STATUS 03
-#define COMMAND_KEEPALIVE 04
-#define COMMAND_DISARM 05
-#define COMMAND_ARM 06
-
-/* response codes */
-#define INIT_RESPONSE 0xAC
-#define READ_FAIL_RESPONSE 0xAF
-#define BAD_COMMAND_RESPONSE 0xBB
-#define RETRACT_RESPONSE 0xE1
-#define EXTEND_RESPONSE 0xE2
-#define KEEPALIVE_RESPONSE 0xE4
-#define DISARM_RESPONSE 0xE5
-#define ARM_RESPONSE 0xE6
-
-// CAMERA FcnCodes
-#define COMMAND_NOOP 0
-#define REQUEST_PACKET_COUNTERS 10
-#define COMMAND_CLEAR_PACKET_COUNTERS 15
-#define REQUEST_ENVIRONMENTAL_DATA 20
-#define REQUEST_POWER_DATA 30
-#define REQUEST_IMU_DATA 40
-#define COMMAND_REBOOT 99
-
-#define PAN_ID 0x0B0B // XBee PAN address (must be the same for all xbees)
-// DO NOT CHANGE without changing for all xbees
+#define XBEE_ADDR 0x0006
```

Comment:
I think these need to be above the include for balloonduino.h otherwise they won't do anything (which is fine in this case, since they are the same values as in the library, but will be confusing to users. Also see comments about possibly passing these into begin